### ISSUE TITLE
Small screen sizes cause the translation dropdown to overflow to the next row.

### ISSUE THREAD

**Issue by alamshafil** [dogecoinfoundation/dogecoin.com#163]
As written by @xanimo in https://github.com/dogecoinfoundation/dogecoin.com/pull/125#pullrequestreview-1057886667

Small screen sizes cause the translation dropdown to overflow to the next row.

> ![image](https://user-images.githubusercontent.com/36016500/182945090-415c2efd-5f90-4400-bf5b-1bb63330abfd.png)
On page load on the landing when the navbar is expanded we're still getting those funky styles on the translation dropdown.

**Comment by Maniixer** [comment]
removing the flag would be a good idea as its not needed to show the languages, Having EN, SP,FR,.... is more then enough.

**Comment by Maniixer** [comment]
I would say removing the flags is best, as some have problem recognizing what flag belongs to what country.

**Comment by Maniixer** [comment]
What you also can do is make the logo smaller on mobile as it does not need to take over the whole nav :p

**Comment by Maniixer** [comment]
![image](https://user-images.githubusercontent.com/92598085/184364645-863fac8d-f848-4908-885e-6bd88e54c513.png)
- changed class names to `class="badge bg-dark text-light comic-neue"` some had class name `class="badge bg-dark text-ligh comic-neuet"`
![image](https://user-images.githubusercontent.com/92598085/184366014-5cfdc7ab-ff12-43f2-a17b-ecd866c09c32.png)

- Added a div to put all the language links in it.
- Removed in .badge{
` padding: 0.35em 1.6500000000000004em;`
}

**Comment by qlpqlp** [comment]
> ![image](https://user-images.githubusercontent.com/92598085/184364645-863fac8d-f848-4908-885e-6bd88e54c513.png)
> 
> * changed class names to `class="badge bg-dark text-light comic-neue"` some had class name `class="badge bg-dark text-ligh comic-neuet"`
>   ![image](https://user-images.githubusercontent.com/92598085/184366014-5cfdc7ab-ff12-43f2-a17b-ecd866c09c32.png)
> * Added a div to put all the language links in it.
> * Removed in .badge{
>   ` padding: 0.35em 1.6500000000000004em;`
>   }

I like your aproche, but I have to admit, shibes like to see the flags also :P

**Comment by Maniixer** [comment]
yes you can keep the flags, if the logo on mobile is just smaller it fixes it.

**Comment by Maniixer** [comment]
@qlpqlp  what you think about this?, before the text would move out of the box because of a fixed width and when hovering over a language it shows it better.
![image](https://user-images.githubusercontent.com/92598085/184544597-a86286f0-b06e-4880-92d3-9808b5d09069.png)
